feed/tests/test_app_response.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from django.db.models.query import QuerySet
+
+
+@pytest.mark.django_db
+def test_feed_app_entrypoint(client):
+    response = client.get("")
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
```

Comment:
This one is not needed,
Here there is no interaction with DB

Reply:
When I try to remove the test access to DB it wont pass because when the test loads the feed page it needs to load posts from DB.

Reply:
thanks